packages/python/m/ci/docker/gh_workflow_multi.py

```diff
@@ -2,8 +2,9 @@
 from .gh_workflow_single import Workflow as DefaultWorkflow
 
 TEMPLATE = """\
-# AUTOGENERATED FILE - Update {m_dir}/m.yaml
-#   then run `m blueprints --update-workflow` to update
+# AUTOGENERATED FILE - Do not update by hand!
+# Edit  {m_dir}/m.yaml then run
```

Comment:
Only one space between `Edit` and `{m_dir}/m.yaml` please

Reply:
> Only one space between `Edit` and `{m_dir}/m.yaml` please

doneski